api/v1alpha1/osbuildconfig_types.go

```diff
@@ -97,6 +97,8 @@ type TargetImage struct {
 	TargetImageType TargetImageType `json:"targetImageType"`
 	// OSTree is the OSTree configuration of the build (optional)
 	OSTree *OSTreeConfig `json:"osTree,omitempty"`
+	// Repositories is the list of additional custom RPM repositories to use when building the image (optional)
+	Repositories *[]Repository `json:"repositorys,omitempty"`
```

Comment:
I think `CustomRepositories` is a better name

Reply:
For the type I disagree. These are Repositories nothing Custom about them.
As for the name, I try to stick as close as possible to the OSBuild terminology. As such and since this field translates to the `Repositories`, I think it should also be names `Repositories`